editor/blocks/list/index.js

```diff
@@ -53,29 +57,57 @@ wp.blocks.registerBlock( 'core/list', {
 			onClick( attributes, setAttributes ) {
 				setAttributes( { align: 'justify' } );
 			}
+		},
+		{
+			icon: 'editor-ul',
+			title: wp.i18n.__( 'Convert to unordered' ),
+			isActive: ( { } ) => false, // To be implemented
```

Comment:
What is to be implemented here? `isActive` isn't considered yet by the editor, but if it were, should this always be `false`? Might be best to simply leave out for now.